repondeur/zam_repondeur/templates/dossiers_add.html

```diff
@@ -32,27 +31,188 @@ <h2>Ajouter un dossier</h2>
         </div>
 
     </form>
 {% endblock %}
 
 {% block scripts %}
-<script src="{{ request.static_url('zam_repondeur:static/jquery/jquery-1.12.4.min.js') }}"></script>
-<script src="{{ request.static_url('zam_repondeur:static/selectize/js/standalone/selectize.min.js') }}"></script>
+<script src="{{ request.static_url('zam_repondeur:static/selectr-2.4.13/selectr.min.js') }}"></script>
 <script type="text/javascript">
     document.addEventListener("DOMContentLoaded", () => {
-        $("#select-dossier").selectize({
-            valueField: "slug",
-            labelField: "titre",
-            searchField: ["titre"],
-            onInitialize: () => {
-                $("#select-dossier-selectized").attr("spellcheck", "false")
+
+        var util = {
+            each: function(a, b, c) {
+                if ("[object Object]" === Object.prototype.toString.call(a)) {
+                    for (var d in a) {
+                        if (Object.prototype.hasOwnProperty.call(a, d)) {
+                            b.call(c, d, a[d], a);
+                        }
+                    }
+                } else {
+                    for (var e = 0, f = a.length; e < f; e++) {
+                        b.call(c, e, a[e], a);
+                    }
+                }
+            },
+            createElement: function(e, a) {
+                var d = document,
+                    el = d.createElement(e);
+                if (a && "[object Object]" === Object.prototype.toString.call(a)) {
+                    var i;
+                    for (i in a)
+                        if (i in el) el[i] = a[i];
+                        else if ("html" === i) el.innerHTML = a[i];
+                        else el.setAttribute(i, a[i]);
+                }
+                return el;
+            },
+            hasClass: function(a, b) {
+                if (a)
+                    return a.classList ? a.classList.contains(b) : !!a.className && !!a.className.match(new RegExp("(\\s|^)" + b + "(\\s|$)"));
+            },
+            addClass: function(a, b) {
+                if (!util.hasClass(a, b)) {
+                    if (a.classList) {
+                        a.classList.add(b);
+                    } else {
+                        a.className = a.className.trim() + " " + b;
+                    }
+                }
+            },
+            removeClass: function(a, b) {
+                if (util.hasClass(a, b)) {
+                    if (a.classList) {
+                        a.classList.remove(b);
+                    } else {
+                        a.className = a.className.replace(new RegExp("(^|\\s)" + b.split(" ").join("|") + "(\\s|$)", "gi"), " ");
+                    }
+                }
+            },
+            includes: function(a, b) {
+                // Poor-man fuzzyness.
+                return b.split(" ").every(n => a.includes(n))
```

Comment:
200 lines of copy-pasta for this single line. There is probably (hopefully?) a better way to monkey patch that.